From aac8e2e3f2e5809de577af4a9f70413e0cf1270a Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Tue, 7 Jun 2022 17:42:26 +0800
Subject: [PATCH] sw64: Kconfig: remove dependence on ARCH_SUPPORTS_ACPI
Patch-mainline: NO, OTHERS
References: openEuler-22.03-LTS
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

SW64 has selected ARCH_SUPPORTS_ACPI by default, so it will always
select ACPI and ACPI_REDUCED_HARDWARE_ONLY.

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/sw_64/Kconfig b/arch/sw_64/Kconfig
index feb8b4218ecb..eebe6c940660 100644
--- a/arch/sw_64/Kconfig
+++ b/arch/sw_64/Kconfig
@@ -87,8 +87,8 @@ config SW64
 	select HAVE_PERF_USER_STACK_DUMP
 	select HAVE_PERF_REGS
 	select ARCH_SUPPORTS_ACPI
-	select ACPI if ARCH_SUPPORTS_ACPI
-	select ACPI_REDUCED_HARDWARE_ONLY if ACPI
+	select ACPI
+	select ACPI_REDUCED_HARDWARE_ONLY
 	select GENERIC_TIME_VSYSCALL
 	select SET_FS
 	select PCI_MSI_ARCH_FALLBACKS
-- 
2.34.1

